lib/utils.js

```diff
@@ -10,18 +10,33 @@ import { fs } from 'appium-support';
 async function killAllSimulators () {
   log.debug('Killing all iOS Simulators');
 
-  // figure out the correct arguments
-  let args = ['-9', '-f'];
+  let appName;
   let xcodeVersion = await getVersion(true);
   if (xcodeVersion.major === 7) {
-    args.push('Simulator');
+    appName = 'Simulator';
   } else {
-    args.push('iOS Simulator');
+    appName = 'iOS Simulator';
   }
 
   try {
-    await exec('pkill', args);
-  } catch (ign) {}
+    await exec('osascript', ['-e', `quit app "${appName}"`]);
```

Comment:
No. Just wondering. Generally you need to allow it to happen. But the Travis VMs might already have it.

You can try locally by going into "System Preferences", then "Security and Privacy", "Privacy", "Accessibility", and unchecking your terminal, or whatever is running the Appium server.